src/System Application/App/Cryptography Management/src/RSAImpl.Codeunit.al

```diff
@@ -164,6 +164,16 @@ codeunit 1476 "RSA Impl." implements "Signature Algorithm v2"
 
 
     #region XmlString
+    [NonDebuggable]
+    procedure ToSecretXmlString(PrivateKey: SecretText; IncludePrivateParameters: Boolean): SecretText
+    var
+        RSAEncryptionHelper: DotNet RSAEncryptionHelper;
+    begin
+        DotNetRSA := DotNetRSA.Create(2048);
```

Comment:
Just call RSA(), as it does exactly this.

Reply:
Nice catch. I should stop reviewing in the browser view. It's hiding too much relevant code. That "constructor" would have been nice to see ;-)